distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -131,6 +132,33 @@ class MyCluster(SpecCluster):
         assert len(cluster.workers) == 2
 
 
+@pytest.mark.asyncio
+async def test_adaptive_killed_worker(cleanup):
+    with dask.config.set({"distributed.deploy.lost-worker-timeout": 1}):
+
+        with SpecCluster(
+            worker={"cls": Nanny, "options": {"nthreads": 1}},
+            scheduler={"cls": Scheduler, "options": {"port": 0}},
+        ) as cluster:
+
+            with Client(cluster) as client:
+
+                cluster.adapt(minimum=1, maximum=1)
+
+                # Scale up a cluster with 1 worker.
+                while len(cluster.workers) != 1:
+                    await asyncio.sleep(1)
+
+                future = client.submit(sleep, 5)
+
+                # Kill the only worker.
+                worker_id = next(iter(cluster.workers))
```

Comment:
If you're feeling fancy ...

```suggestion
                [worker_id] = cluster.workers
```